content/blog/2016/2016-01-27-jenkins-world-call-for-papers.adoc

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: "Jenkins World 2016: Call For Papers Is Open!"
+tags:
+- jenkins world
+- event
+- jenkins
+author: alyssat
+---
+
+image:/images/conferences/Jenkins-World_125x125.png[Jenkins World 2016,180,float="right",link=" https://jenkins-cfp.herokuapp.com/events/jenkins-world-2016"]
+
+Planning is underway for the annual Jenkins User Conference. Conference dates are September 13 - 15, 2016, at the Santa Clara Convention Center in California.  In previous years, Jenkins User Conference has been a multi-city conference. With year after year growth, the Jenkins User Conference has proven that Jenkins is most popular open source automation server.  That being said, Jenkins User Conference will be Jenkins World in 2016. It will be a one city, three days, one global conference.

```

Comment:
This paragraph needs some work, The number of times "conference" appears makes it read funny to me.